packages/instance-model/lib/model.js

```diff
@@ -254,6 +254,7 @@ const InstanceModel = AmpersandModel.extend(
 
         this.set({ refreshingStatus: 'ready', refreshingStatusError: null });
       } catch (err) {
+        console.log("refreshingStatusError!", err.stack)
```

Comment:
Do we want to `log` these errors?

Reply:
Yeah, this is definitely a debugging leftover :) I think we don’t quite need to do that here because we already log failed driver commands in general, and errors here will typically fall into that category, but we can look into adding it if this does turn out to be a problem